packages/acceptance-tests/pkg-tests-core/sources/utils/makeTemporaryEnv.ts

```diff
@@ -20,9 +20,9 @@ global.makeTemporaryEnv = generatePkgDriver({
     [command, ...args],
     {cwd, projectFolder, registryUrl, env, ...config},
   ) {
-    const rcEnv = {};
-    for (const [key, value] of Object.entries(config))
-      rcEnv[`YARN_${key.replace(/([A-Z])/g, `_$1`).toUpperCase()}`] = value;
+    const rcEnv: Record<string, any> = {};
+    for (const key of Object.keys(config))
+      rcEnv[`YARN_${key.replace(/([A-Z])/g, `_$1`).toUpperCase()}`] = config[key];
```

Comment:
I'm curious, why not `Object.entries`?

Reply:
Because TS compiler had problems with recognizing `Object.entries` despite `es2017` was in the `tsconfig.json -> lib`. Since it was the only place, I did't start to dig deeper, just switched to `Object.keys`